pkg/openthread/contrib/netdev/openthread_netdev.c

```diff
@@ -78,6 +89,16 @@ static void *_openthread_event_loop(void *arg)
     diagInit(sInstance);
 #endif
 
+    /* Init default parameters */
```

Comment:
Stupid question: would it make sense to add these init calls in the openthread_bootstrap function ?